packages/app-desktop/gui/styles/dialog-modal-layer.scss

```diff
@@ -32,13 +32,16 @@
 	}
 
 	&.-fullscreen {
+		max-width: 100vw;
+		max-height: 100vh;
+
 		&::backdrop {
 			background-color: var(--joplin-background-color);
 		}
 
 		> .content {
-			width: calc(100% - 20px);
-			padding: 10px;
+			margin: 0;
+			padding: 0;
```

Comment:
These changes allow the "change application layout" screen to completely overlay the main app content. At present,
- The application panels are shown **behind** the application layout `<dialog>`. The modal `<dialog>` prevents these panels from receiving keyboard focus while changing the application layout.
- The layout `<dialog>` contains the movement controls and a resizable region for each panel.